docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss

```diff
@@ -181,3 +181,17 @@
     background-image: linear-gradient(to $icon-position, var(--palette--tertiary-highlight) $icon-width, transparent $icon-width);
   }
 }
+
+@mixin hb-well {
+  @include hb-themes(('colorful', 'airy')) {
+    @include hb-global-color('background-color', 'gray-light');
+  }
+
+  @include hb-traditional {
+    @include hb-pairing-color('background-color', 'tertiary-highlight');
+  }
+
+  padding: hb-calculate-rems(20px) hb-calculate-rems(18px);
+  border: 0 none;
+  display: table;
```

Comment:
what is display: table for?

Reply:
@kaseybon maybe you know?

Reply:
This is what makes the breakout box work with an image is floated left or right nearby. Without it the background bleeds behind the image and looks broken.